src/import-data/protected-data-utils.ts

```diff
@@ -68,12 +68,27 @@ export const isProtectedSource = (importRequest: ImportRequest): boolean => {
   }
 };
 
-// This method identifies whether a workspace qualifies as protected.
-// 'Protected' here means that it has enhanced logging - either on its own or because it has an auth domain.
-// For now this also means only GCP workspaces are included.
+/**
+ * Determine whether a workspace is considred protected.
+ *
+ * For Azure workspaces, this checks for the "protected-data" policy.
+ * For Google workspaces, this checks for has enhanced logging - either directly or from an auth domain.
+ *
+ * @param workspace - The workspace.
+ */
 export const isProtectedWorkspace = (workspace: WorkspaceWrapper): boolean => {
```

Comment:
@nawatts did you see that there is also a method to determine this in `workspace-utils`?

```
export const hasProtectedData = (workspace: BaseWorkspace): boolean => containsProtectedDataPolicy(workspace.policies);

export const containsProtectedDataPolicy = (policies: WorkspacePolicy[] | undefined): boolean =>
  _.any((policy) => policy.namespace === 'terra' && policy.name === 'protected-data', policies);

Reply:
It seems like you could at least refactor yours to call `containsProtectedDataPolicy` so that we don't duplicate the logic of the specific policy name.

Reply:
No, I missed that.

Reply:
Done in #4344.